change/@fluentui-react-motion-f8cc73b4-883a-4c0a-a9df-8df610c222ff.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31698&commitId=1f885cfbc0ea498990e766628399345b5f827489&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|3| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|3| Changed|

</details>


  <details><summary><b>Card Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged.appearance - High Contrast.normal.chromium.png|0| Added|
|Card Converged.appearance - RTL.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Card Converged - Interactive</b> 9 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Outline - High Contrast.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle - Dark Mode.normal.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.click.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.hover.chromium.png|0| Added|
|Card Converged - Interactive.appearance interactive - Subtle.normal.chromium.png|0| Added|

</details>


  <details><summary><b>Image Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Variations Fallback.normal.chromium.png|0| Added|

</details>


  <details><summary><b>SwatchPicker Converged</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|SwatchPicker Converged.spacing.default.chromium.png|0| Added|
|SwatchPicker Converged.Shape.default.chromium.png|0| Removed|
|SwatchPicker Converged.Spacing.default.chromium.png|0| Removed|
|SwatchPicker Converged.default.default.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>